docs/general/manage-tokens/how-to-transfer-rose-into-evm-paratime.mdx

```diff
@@ -7,27 +7,31 @@ description: >-
 import DocCard from '@theme/DocCard';
 import {findSidebarItem} from '@site/src/sidebarUtils';
 
-# How to Transfer ROSE into Emerald ParaTime
+# How to Transfer ROSE into an EVM ParaTime
 
 ## About
 
-[Emerald](../developer-resources/emerald-paratime/README.mdx) is an EVM-compatible blockchain running inside the Oasis ParaTime. Because the balance of your ROSE wallet is stored on the consensus layer - outside of ParaTime's reach - there are special mechanisms for Emerald to access your tokens (formally defined in [ADR 0003](/adrs/0003-consensus-runtime-token-transfer)).
+An EVM-compatible ParaTime is one that integrates the Oasis Runtime SDK's EVM module.
```

Comment:
These are dev docs. If you end up on a JS bundler page but don't know what JS is, are you really in the right place? I'll add a clause about it, but I'd prefer to keep these docs concise rather than maintain content better found elsewhere.